src/inc/redirect/Mlp_Language_Negotiation.php

```diff
@@ -109,10 +110,11 @@ private function get_possible_matches( Array $translations ) {
 	private function get_fallback_match() {
 
 		return array (
-			'priority' => 0,
-			'url'      => '',
-			'language' => '',
-			'site_id'  => 0
+			'priority'          => 0,
+			'url'               => '',
+			'language'          => '',
+			'site_id'           => 0,
+			'target_content_id' => 0,
```

Comment:
No, `content_id` is good, because it's not only used for posts. If you visit a term archive, the ID refers to the term taxonomy ID, for example.

Thanks for updating the PR. Could you take care of the indentation in the two arrays, then I'll merge this right away.